test/signs_ui/alerts/state_test.exs

```diff
@@ -12,6 +12,33 @@ defmodule SignsUi.Alerts.StateTest do
     end
   end
 
+  describe "handle_call :active_alert_ids" do
+    test "returns the alert_ids out of the state" do
+      state = %SignsUi.Alerts.State{
+        alerts: %{
+          # fill in sample data here, but it's of this shape:
```

Comment:
No need for this comment. I left it for you, to add more sample data, but actually I think these three alerts are fine.